js/dashboard.js

```diff
@@ -106,7 +108,13 @@ var Dashboard = {
       // retieve cards content by ajax
       if (Dashboard.ajax_cards) {
          Dashboard.getCardsAjax();
-      } else {
+      }
+
+      // init filters from storage
+      Dashboard.initFilters();
```

Comment:
I would have done that before `Dashboard.getCardsAjax();`.

Reply:
You're completely right